compiler/src/iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -970,6 +1002,12 @@ static bool isFusableWithProducer(OpOperand &operand, bool aggressiveFusion) {
   Operation *producer = operand.get().getDefiningOp();
   Operation *consumer = operand.getOwner();
 
+  // Fuse linalg ops with set encoding op if the operand is an `outs` value.
+  if (isa<linalg::LinalgOp>(consumer) &&
+      isa<IREE::LinalgExt::SetEncodingOp>(producer)) {
+    return cast<DestinationStyleOpInterface>(consumer).isDpsInit(&operand);
+  }
```

Comment:
I could be wrong. Why fuse linalg_ext.set_encoding + linalg? Shouldn't we fuse linalg + linalg_ext.set_encoding?

Should this be

```suggestion
  if (isa<linalg::LinalgOp>(producer) &&
      isa<IREE::LinalgExt::SetEncodingOp>(consumer)) {
    return cast<DestinationStyleOpInterface>(producer).isDpsInit(&operand);
  }
```

Reply:
Removed this part for now. We might need to revisit this, but not needed for the initial landing of this.